flight/PiOS/ChibiOS/os/common/startup/ARMCMx/compilers/GCC/crt0_v7m.S

```diff
@@ -120,7 +120,7 @@
  * @brief   RAM areas initialization switch.
  */
 #if !defined(CRT0_INIT_RAM_AREAS) || defined(__DOXYGEN__)
-#define CRT0_INIT_RAM_AREAS                 TRUE
+#define CRT0_INIT_RAM_AREAS                 FALSE
```

Comment:
Please don't do this in ChibiOS source... define it somewhere in our sources.

Reply:
I think this is chconf kinda stuff?

Reply:
Yep, although I can't find that... did we get rid of it?

Reply:
PiOS/STM32/inc/chconf.h (a per-target one takes precedence but isn't necessary)

Reply:
Oh yep. That's good. :+1:

Reply:
OK, I'm gonna put it there. In regards to chconf.h, If I'm getting this to work, I'll probably define a bunch of macros so that PiOS compiles both on ChibiOS 2.6.6 on F3 and below and 17.6.3 on F4, and eventually F7, if we go there. SPRF3e is probably gonna blow again with new ChibiOS.

Reply:
Why use a different version on F3 and below?

Reply:
I'm gonna give it a try, but SPRF3e is really damn short on space in the firmware image. Chances are that it breaks compilation again. Plus testing PipXtreme and FlyingPio is kind of an issue for me. :)